src/services/goals.js

```diff
@@ -545,6 +545,7 @@ function reduceGoals(goals, forReport = false) {
         currentValue.objectives,
       ),
       isNew: false,
+      endDate: currentValue.endDate,
```

Comment:
@AdamAdHocTeam 

[From sequelize docs](https://sequelize.org/api/v6/class/src/model.js~model):

> ...if getters and/or setters are defined for field they will be invoked, instead of returning the value from dataValues

Before we were grabbing endDate as part of ```...currentValue.dataValues``` in this reducer, which meant the custom getter wasn't being called. Turns out all we have to do is access it as above and that get function I wrote on the model is invoked. Good call!